app/src/main/java/com/nextcloud/talk/conversationinfo/viewmodel/ConversationInfoViewModel.kt

```diff
@@ -200,6 +202,56 @@ class ConversationInfoViewModel @Inject constructor(
             })
     }
 
+    fun archiveConversation(user: User, token: String) {
```

Comment:
Nothing to implement for now (can be done in followup PR if you are interested), but just as a sidenote:

It would be nice to have the archive/unarchive button also available in the conversations context menu. This would lead to the question how duplicated code can be avoided to call the repository. 
In the long term we should use "UseCases" for it to make this more reusable. See explanations at https://medium.com/@ami0275/mvvm-clean-architecture-pattern-in-android-with-use-cases-eff7edc2ef76

But as said, should be done later on and not in this PR. So for now i suggest to don't implement it in the context menu.